app/controllers/application_controller.rb

```diff
@@ -1499,11 +1499,11 @@ def set_config(db_record) # pass in the db record, either @host or @vm
       @account_policy.push(:field       => "Reset Lockout Counter",
                            :description => db_record.operating_system.reset_lockout_counter) unless db_record.operating_system.reset_lockout_counter.nil?
     end
-    if db_record.respond_to?("vmm_vendor") # For Host table, this will pull the VMM fields
+    if db_record.respond_to?("vmm_vendor_display") # For Host table, this will pull the VMM fields
       @vmminfo = []    # This will be an array of hashes to allow the rhtml to pull out each field by name
 
       @vmminfo.push(:vmminfo     => "Vendor",
-                    :description => db_record.vmm_vendor) unless db_record.vmm_vendor.nil?
+                    :description => db_record.vmm_vendor_display) unless db_record.vmm_vendor_display.nil? # This would never have been nil
```

Comment:
If the record is valid, then this can't be `nil`. Am I safe just to remove this condition?